packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -181,5 +181,119 @@
             "throw new ${1:Exception}($0)"
         ],
         "description": "Throw a new exception"
+    },
+    "list": {
+        "prefix": "list<",
+        "body": [
+            "list<${1:object}> ${2:listName} = new list<$1>();"
+        ],
+        "description": "list constructor"
+    },
+    "map": {
+        "prefix": "map<",
```

Comment:
I wonder if the `<` is necessary. Would we potentially have `map` and `map<`. Seems unlikely.